dChatServer/ChatPacketHandler.cpp

```diff
@@ -337,121 +337,142 @@ void ChatPacketHandler::HandleRemoveFriend(Packet* packet) {
 
 void ChatPacketHandler::HandleChatMessage(Packet* packet) {
 	CINSTREAM_SKIP_HEADER;
-	LWOOBJID playerID = LWOOBJID_EMPTY;
+	LWOOBJID playerID;
 	inStream.Read(playerID);
 
 	const auto& sender = Game::playerContainer.GetPlayerData(playerID);
+	if (!sender || sender.GetIsMuted()) return;
 
-	if (!sender) return;
-
-	if (sender.GetIsMuted()) return;
-
-	inStream.SetReadOffset(0x14 * 8);
-
-	uint8_t channel = 0;
+	eChatChannel channel;
+	uint32_t size;
+	
+	inStream.IgnoreBytes(4);
 	inStream.Read(channel);
+	inStream.Read(size);
+	inStream.IgnoreBytes(77);
```

Comment:
Why 77?  This should be commented and elaborated on for future maintainers otherwise we'll RE this again.

Reply:
there's no data there, so I'll make a comment to indicate as such